templates/.snapshots/TestDevspaceYaml-devspace.yaml.tpl-devspace.yaml.snapshot

```diff
@@ -363,6 +373,48 @@ profiles:
             name: E2E
             value: "true"
 
+  - name: binarySync
+    description: Synchronizes just content of bin folder and don't do any build related stuff in the devspace pod
+    activation:
+      - vars:
+          DEVENV_SYNC_BINARIES: "true"
+    patches:
+      - op: replace
+        path: dev.app.devImage
+        value: gcr.io/outreach-docker/bootstrap/dev-slim:stable
+      - op: replace
+        path: dev.app.sync
+        value:
+          - path: ./bin:${DEV_CONTAINER_WORKDIR}
+            printLogs: true
+            disableDownload: true
+            waitInitialSync: true
+      - op: add
+        path: dev.app.patches
+        value: 
+          op: replace
+          path: spec.containers[0].resources.limits
+          value:
+            cpu: 200m
+            memory: 400Mi
+      - op: add
+        path: hooks
+        value:
+          name: reset-dev
+          events: ["devCommand:after:execute"]
+          command: |-
+            "${DEVENV_DEVSPACE_BIN}" reset pods -s
+    merge: 
+      dev:
+        app:
+          # https://www.devspace.sh/docs/configuration/dev/#dev-terminal
+          terminal:
+            enabled: true
+            disableReplace: true
+            workDir: ${DEV_CONTAINER_WORKDIR}
+            command: |-
```

Comment:
My understanding is that this profile will created a devspace pod, copy the local binary over, and then leave a terminal running for engineers to interact with/run their service/debug. Is that the intended goal? If so, we should be abundantly cautious around setting strict limits that will apply to 100% of services.

```
      - op: add
        path: dev.app.patches
        value: 
          op: replace
          path: spec.containers[0].resources.limits
          value:
            cpu: 200m
            memory: 400Mi
```

The CPU is compressible, so I'm less concerned. It might just cause the pod to take longer or probes to timeout. I'm concerned that some binaries will _always_ take more than 400Mi when the service is running and OOMKill and our support load will increase.

Reply:
Yeah, I agree. However how to ensure, that we have at least 200Mi of memory? Probably somehow parse the limit and replace it when its lower than 200MB. I'll try that.

Reply:
which will be challenging thanks to that lovely "Mi"/"Gi" suffixes

Reply:
I'm in favor of leaving limits up to teams.

Reply:
Removed this part